app/controllers/moves_controller.rb

```diff
@@ -13,7 +13,8 @@ def index
   def show; end
 
   def new
-    @move = Move.new(moveable_type: 'Server')
+    moveable_id = Server.friendly.select(:id).find(params[:server_id]).id if params[:server_id]
+    @move = Move.new(moveable_type: "Server", moveable_id: moveable_id)
```

Comment:
```suggestion
    @move = Move.new(moveable_type: "Server", moveable_id: moveable_id)
    @move.moveable = Server.friendly.select(:id).find(params[:server_id]).id if params[:server_id].present?
```

Also maybe we should use `moveable_id` in query param?